src/patterns/ec2-app/base.ts

```diff
@@ -348,6 +348,13 @@ export class GuEc2App {
     // TODO: Do we need to tag all resources with this value? What would the use-cases be?
     Tags.of(autoScalingGroup).add(TagKeys.PATTERN_NAME, this.constructor.name, { applyToLaunchedInstances: true });
 
+    // This allows automatic shipping of instance Cloud Init logs when using the
+    // `cdk-base` Amigo role on your AMI.
+    Tags.of(autoScalingGroup).add(
+      "LogKinesisStreamName",
```

Comment:
Could we move this string into [`TagKeys`](https://github.com/guardian/cdk/blob/main/src/constants/tag-keys.ts)?